identity-and-access-management/secret-manager/quickstart.mdx

```diff
@@ -6,11 +6,11 @@ content:
   h1: Quickstart
   paragraph: This page shows you how to get started with Scaleway's Secret Manager.
 dates:
-  validation: 2023-06-16
+  validation: 2023-11-21
   posted: 2023-02-21
 ---
 
-Scaleway’s [Secret Manager](/identity-and-access-management/secret-manager/concepts/#secret-manager) allows you to conveniently [store, access and share sensitive data](/identity-and-access-management/secret-manager/reference-content/data-encryption-with-secret-manager/) such as passwords, API keys and certificates. In this quickstart, we show you how to create and manage a secret, how to create and manage a secret's version and how to delete your resources once you are done using them.
+In this quickstart, we show you how to create a [folder](/identity-and-access-management/secret-manager/concepts/#folder) and a [secret](/identity-and-access-management/secret-manager/concepts/#secret) and how to create a secret's [version](/identity-and-access-management/secret-manager/concepts/#version).
 
 <Macro id="iam-requirements" />
 
```

Comment:
Again, the requirement is not clear. Since the use case is based on console, I don't hink we need to mention any API key